python/tests/integration/connectors/memory/test_azure_cosmosdb_memory_store.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from datetime import datetime
+
+import numpy as np
+import pytest
+
+from semantic_kernel.connectors.memory.azure_cosmosdb.azure_cosmos_db_memory_store import (
+    AzureCosmosDBMemoryStore,
+)
+from semantic_kernel.memory.memory_record import MemoryRecord
+from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+
+
+def create_embedding(non_zero_pos: int) -> np.ndarray:
+    # Create a NumPy array with a single non-zero value of dimension 1546
+    embedding = np.zeros(1536)
+    embedding[non_zero_pos - 1] = 1.0
+    return embedding
+
+
+@pytest.fixture
+def memory_record1():
+    return MemoryRecord(
+        id="test_id1",
+        text="sample text1",
+        is_reference=False,
+        embedding=create_embedding(non_zero_pos=1),
+        description="description",
+        additional_metadata="additional metadata",
+        external_source_name="external source",
+        timestamp=datetime.now(),
+    )
+
+
+@pytest.fixture
+def memory_record2():
+    return MemoryRecord(
+        id="test_id2",
+        text="sample text2",
+        is_reference=False,
+        embedding=create_embedding(non_zero_pos=2),
+        description="description",
+        additional_metadata="additional metadata",
+        external_source_name="external source",
+        timestamp=datetime.now(),
+    )
+
+
+@pytest.fixture
+def memory_record3():
+    return MemoryRecord(
+        id="test_id3",
+        text="sample text3",
+        is_reference=False,
+        embedding=create_embedding(non_zero_pos=3),
+        description="description",
+        additional_metadata="additional metadata",
+        external_source_name="external source",
+        timestamp=datetime.now(),
+    )
+
+
+async def azurecosmosdb_memorystore() -> MemoryStoreBase:
+    store = await AzureCosmosDBMemoryStore.create(
+        num_lists=1, similarity="COS", vector_dimensions=8
+    )
+    return store
+
+
+@pytest.mark.asyncio
+async def test_get_collection_async():
+    store = await azurecosmosdb_memorystore()
+    result = await store.get_collections_async()
+    assert "semanticKernelTesting" in result
```

Comment:
This test requires to have `semanticKernelTesting` container in test database, which can be improved. Following my previous comment regarding collections handling, it would be better to implement `create_collection_async` method which currently is empty, and then include test that will:
1. Create collection (in this case - container).
2. Fetch collection and verifies that created collection exists.
3. Clean-up - remove created collection.

Reply:
Added a test case for this.